doc/manual/manual.tex

```diff
@@ -7021,6 +7021,158 @@ \subsubsection{Using reconstructed surface velocities by GPlates}
+we also use harmonic averaging of the material properties.
+\lstinputlisting[language=prmfile]{cookbooks/burnman/material_model.part.prm}
+As the reference profile has a depth dependent density and also contains data for the compressibility, 
+this material model supports compressible convection models. 
+
+For the adiabatic conditions and the gravity model, we also specify that we want to use the respective
+\texttt{ascii data} plugin, and provide the data directory in the same way as for the material 
+model. The gravity model automatically uses the same file as the adiabatic conditions model.
+\lstinputlisting[language=prmfile]{cookbooks/burnman/adiabatic_conditions.part.prm}
+\lstinputlisting[language=prmfile]{cookbooks/burnman/gravity_model.part.prm}
+
+To make use of the reference state we just imported from BurnMan, we choose a formulation of the 
+equations that employs a reference state and compressible convection, in this case the anelastic 
+liquid approximation (see Section~\ref{sec:ala}). 
+\lstinputlisting[language=prmfile]{cookbooks/burnman/formulation.part.prm}
+This means that the reference profiles are used for all material properties in the model, except for
+the density in the buoyancy term (on the right-hand side of the force balance equation~\eqref{eq:stokes-1}, 
+which in the limit of the anelastic liquid approximation becomes Equation~\eqref{eq:stokes-ALA-1}). 
+In addition, the density derivative in the mass conservation equation 
+(see Section~\ref{sec:mass-conservation-approximation}) is taken from the adiabatic 
+conditions, where it is computed as depth derivative of the provided reference density profile
+(see also Section~\ref{sec:combined_formulations}).
+
+\paragraph{Visualizing the model output.}
+If we look at the output of our model (for example in \texttt{ParaView}), we can see how cold, highly 
+viscous slabs are subducted and hot plumes rise from the core-mantle boundary. The final time step of 
+the model is shown in Figure~\ref{fig:burnman-convection}, and the full model evolution can be found 
+at \url{https://youtu.be/nRBOpw5kp-4}.
+Visualizing material properties such as density, thermal expansivity or specific heat shows how they
+change with depth, and reveals abrupt jumps at the phase transitions, where properties change from one
+mineral phase to the next. We can also visualize the gravity and the adiabatic profile, to ensure that
+the data we provided in the \url{data/adiabatic-conditions/ascii-data/isentrope_properties.txt} file
+is used in our model. 
+
+\begin{figure}
+  \includegraphics[width=0.48\textwidth]{cookbooks/burnman/temperature}
+  \hfill
+  \includegraphics[width=0.48\textwidth]{cookbooks/burnman/viscosity}
+  \caption{\it Compressible convection in a 2d spherical shell, using a reference profile exported 
+               form BurnMan, which is based on the Birch-Murnaghan equation of state. Shown is the
+               state at the end of the model evolution over 260\,Ma.}
```

Comment:
In my zeal to eradicate the passive voice, let me suggest to change the last sentence to "The figure shows the state at the end....".